### PR TITLE
Translate map screen

### LINKED ISSUE #408: English translation for map screen
## Overview (Required)
- Probably we can use this translation
https://droidkaigi.jp/2018/en/

### PR DESCRIPTION
**Pull request by ycrack**
## Issue
- close #408 

## Overview (Required)
- Translate strings

## Addition
Sorry for dirty commits. I'm looking for solve this problem.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -118,5 +115,8 @@
     <string name="about_contributors_description">Those who contribute to creation.</string>
     <string name="about_version_title">Version</string>
     <string name="about_version_description">%1$s</string>
+    <string name="map_meeting_rooms_name">DroidKaigi 2018 Place</string>
+    <string name="map_address">Sumitomo Fudosan Shinjuku Grand Tower 5F 8-17-3 Nishi Shinjuku, Shinjuku City, Tokyo 160-0023</string>
+    <string name="map_nearby_stations">Marunouchi subway line: 3 minutes walk from Nishi Shinjuku Station Gate No.1 Toei Oedo subway line: 7 minutes walk from Tochomae Station Gate E4 Toei Oedo subway line: 11 minutes walk from Shinjuku-Nishiguchi Station Gate D4 JR lines, Marunouchi/Toei Shinjuku/Toei Oedo subway line, Odakyu line, Keio line: 15 minutes walk from Shinjuku Station West Gate</string>
```
Comment (konifar) [inline app/src/main/res/values/strings.xml]: Can you move them to `<!-- Map -->` section?

**Comment by konifar** [comment]
Thanks for many contribution! 👍 
I commented one.

**Comment by konifar** [comment]
LGTM! Thanks for quick fix ⚡️

**Comment by ycrack** [comment]
👍

**Pull request merged**